app/views/accessibility.njk

```diff
@@ -0,0 +1,76 @@
+      </p>
+      <ul class="govuk-list govuk-list--bullet">
+        <li> email <a class="govuk-link" href="mailto:FTF@rpa.gov.uk">FTF@rpa.gov.uk </a></li>
+        <li> call 0300 0200 301 and follow the options for the Farming Transformation Fund scheme </li>
+      </ul>
+      <p class="govuk-body">We’ll consider your request and get back to you in 10 days.</p>
+      <h2 class="govuk-heading-m">Reporting accessibility problems with this service</h2>
+      <p class="govuk-body">We’re always looking to improve the accessibility of this service. If you find any problems not listed on this page or think we’re not meeting accessibility requirements:</p>
+      <ul class="govuk-list govuk-list--bullet">
+        <li> email <a class="govuk-link" href="mailto:FTF@rpa.gov.uk">FTF@rpa.gov.uk</a></li>
+        <li> call 0300 0200 301 and follow the options for the Farming Transformation Fund scheme </li>
+      </ul>
+      <h2 class="govuk-heading-m">Enforcement procedure</h2>
+      <p class="govuk-body">
+        The Equality and Human Rights Commission (EHRC) is responsible for enforcing the Public Sector Bodies (Websites and Mobile Applications) (No. 2) Accessibility Regulations 2018 (the ‘accessibility regulations’). If you’re not happy with how we respond to your complaint, <a class="govuk-link" href="https://www.equalityadvisoryservice.com/">contact the Equality Advisory and Support Service (EASS).</a>
+        </p>
+
+      <h2 class="govuk-heading-l">Contacting us by phone or visiting us in person<h2>
+      <p class="govuk-body">
+        We provide a text relay service for people who are D/deaf, hearing impaired or have a speech impediment.<br/> <br/>
+        Our offices have audio induction loops, or if you contact us before your visit we can arrange a British Sign Language (BSL) interpreter.<br/><br/>
+        To contact us:
+      </p>
+      <ul class="govuk-list govuk-list--bullet">
+        <li> email <a class="govuk-link" href="mailto:FTF@rpa.gov.uk">FTF@rpa.gov.uk </a></li>
+        <li> call 0300 0200 301 and follow the options for the Farming Transformation Fund scheme </li>
+      </ul>
+
+      <h2 class="govuk-heading-l">Technical information about this service’s accessibility</h2>
+      <p class="govuk-body">
+        RPA is committed to making its website accessible, in accordance with the Public Sector Bodies (Websites and Mobile Applications) (No. 2) Accessibility Regulations 2018.
+      </p>
+      <h2 class="govuk-heading-m">Compliance status </h2>
+      <p class="govuk-body">
+        This website is fully compliant with the <a class="govuk-link" href="https://www.w3.org/TR/WCAG21">Web Content Accessibility Guidelines version 2.1</a> AA standard.
+      </p>
+      <h2 class="govuk-heading-l">Preparation of this accessibility statement </h2>
+      <p class="govuk-body">
+        This statement was prepared on 5 August 2021.<br/><br/>
```

Comment:
I know this is only a place holder but can these dates be replaced with `TBC` or `TODO` or something to highlight that we will need to update it in the future.